### PR TITLE
change  "ripristino di fabbrica" to "factory default"

### PR DESCRIPTION
**Pull request by cotosso**
I propose to change "ripristino completo impostazioni di fabbrica" with  " factory default"
it takes up less space on the page and is now a common term even in the Italian language
image

### CODE REVIEW

**Comment by cotosso** [comment]
"Factory reset" (as for english translation) could even be better, what you think @andre8244 ?

**Comment by andre8244** [comment]
> "Factory reset" (as for english translation) could even be better, what do you think @andre8244 ?

Yes, I think "factory reset" is better that "factory default"

**Comment by cotosso** [comment]
> > "Factory reset" (as for english translation) could even be better, what do you think @andre8244 ?
> 
> Yes, I think "factory reset" is better that "factory default"

@andre8244  I substitute "default" with "reset"

File: public/i18n/it/translation.json [hunk 1]
```diff
@@ -1143,14 +1143,14 @@
       "update": "Aggiornare"
     },
     "factory_reset": {
-      "title": "Ripristino completo impostazioni di fabbrica",
+      "title": "Factory reset",
       "perform_factory_confirm": "Confermare il ripristino",
```
Comment (andre8244) [inline public/i18n/it/translation.json:1147]:
```suggestion
      "perform_factory_confirm": "Conferma factory reset",
```

File: public/i18n/it/translation.json [hunk 2]
```diff
@@ -1143,14 +1143,14 @@
...
-      "perform_factory_reset": "Eseguire il reset di fabbrica",
+      "perform_factory_reset": "Eseguire factory reset",
```
Comment (andre8244) [inline public/i18n/it/translation.json:1148]:
```suggestion
      "perform_factory_reset": "Esegui factory reset",
```

File: public/i18n/it/translation.json [hunk 3]
```diff
@@ -1143,14 +1143,14 @@
...
       "current_version": "Versione attuale",
       "resetting_description": "Ripristino e riavvio dell'unità in corso, si prega di attendere...",
-      "description_helper": "Il reset di fabbrica ripristina la versione attualmente installata.",
-      "perform_factory_reset_description": "Il reset di fabbrica elimina definitivamente tutti i dati e le impostazioni.",
-      "description": "Ripristino di fabbrica elimina tutti i pacchetti installati e le impostazioni personalizzate, riportando il firewall al suo stato originale come era dopo l'installazione.",
+      "description_helper": "Il factory reset ripristina la versione attualmente installata.",
+      "perform_factory_reset_description": "Il factory reset elimina definitivamente tutti i dati e le impostazioni.",
+      "description": "Factory reset elimina tutti i pacchetti installati e le impostazioni personalizzate, riportando il firewall al suo stato originale come era dopo l'installazione.",
```
Comment (andre8244) [inline public/i18n/it/translation.json:1153]:
```suggestion
      "description": "Il factory reset elimina tutti i pacchetti installati e le impostazioni personalizzate, riportando il firewall al suo stato originale come era dopo l'installazione.",
```

**Review by andre8244 — APPROVED** [review]

**Pull request merged**